lib/ansible/modules/network/onyx/onyx_syslog.py

```diff
@@ -0,0 +1,424 @@
+        syslog_config = self._get_syslog_config()
+        self._current_config = dict()
+        current_config = dict()
+
+        map_key_value = {
+            'Log format': 'format',
+            'Allow receiving of messages from remote hosts': 'received',
+            'Subsecond timestamp field': 'seconds_enabled',
+            'Local logging level': 'local_level',
+            'Default remote logging level': 'trap'
+        }
+
+        map_key_list = {
+            'Levels at which messages are logged': {
+                'alias': {
+                    'CLI commands': 'cli_level',
+                    'Audit messages': 'audit_level'
+                }
+            },
+            'Levels at which messages are echoed to all sessions': {
+                'name': 'monitor',
+                'keys': ['monitor_facility', 'monitor_level']
+            }
+        }
+        override_prefix = 'Override for class'
+        for obj in syslog_config:
+            for key, value in obj.items():
+                if key in map_key_value:
+                    current_key = map_key_value[key]
+                    current_config[current_key] = value
+                elif key in map_key_list:
+                    map_obj = value[0]
+                    map_key_element = map_key_list[key]
+                    if 'name' in map_key_element:
+                        name = map_key_element['name']
+                        keys = map_key_element['keys']
+                        current_config[name] = [{keys[0]: key, keys[1]: value}
+                                                for key, value in map_obj.items()]
+                    else:
+                        alias = map_key_element['alias']
+                        for show_key, current_key in alias.items():
+                            current_config[current_key] = map_obj[show_key]
+                elif override_prefix in key:
+                    override_class = key.replace(override_prefix, '').strip()
+                    override_priority = value
+                    if 'local_override'in current_config:
+                        current_config['local_override'].append({
+                            'override_class': override_class,
+                            'override_priority': override_priority
+                        })
+                    else:
+                        current_config['local_override'] = [{
+                            'override_class': override_class,
+                            'override_priority': override_priority
+                        }]
+
+        current_config['received'] = True if current_config.get('received') == 'yes' else False
+        ''' extract 'format/fields' from running config line (if any ?) its single line - no'''
+        if syslog_config and syslog_config[0].get('lines'):
+            lines = syslog_config[0].get('lines')
+            for line in lines:
+                match = self.FORMAT_WELF_RGX.match(line)
+                if match:
```

Comment:
```python
if match:
    current_config['format_welf_host'] = match.group(1)
    continue
```
no need to proceed to other checks when you find a match
do it for all match check